crates/isograph_config/src/compilation_options.rs

```diff
@@ -80,9 +81,13 @@ impl Default for OptionalValidationLevel {
     }
 }
 
-#[derive(Deserialize)]
+#[derive(Deserialize, JsonSchema)]
 #[serde(deny_unknown_fields)]
-struct ConfigFile {
+pub struct ConfigFile {
```

Comment:
this causes `title: "ConfigFile"` in the schema file, can we change that to IsographProjectConfig or something?